docroot/themes/humsci/humsci_basic/src/scss/utilities/_raised-cards.scss

```diff
@@ -84,3 +84,88 @@
     }
   }
 }
+
+// Dark Pattern and Inversion settings for hb-card, placed here because of the order of the classes when applied
+.hb-raised-cards .hb-dark-pattern .hb-card,
+.hb-dark-inversion .hb-raised-cards .views-element-container .hb-card {
```

Comment:
I took a look at how the class flow in the markdown. What you have here makes sense. I can't think of any other way to inject the styles within the code above. 👍

Reply:
Thank you.